autosubmit/autosubmit.py

```diff
@@ -1381,8 +1382,14 @@ def expid(description, hpc="", copy_id='', dummy=False, minimal_configuration=Fa
             copy_id_folder = os.path.join(root_folder, copy_id)
             if not os.path.exists(copy_id_folder):
                 raise AutosubmitCritical(f"Experiment {copy_id} doesn't exists", 7011)
+            if minimal_configuration:
+                conf_dir = Path(copy_id_folder) / "conf"
+                if not list(conf_dir.glob('minimal.y*')):
+                    raise AutosubmitCritical(
+                            "Cannot copy experiment that does not have a minimal.yml file", 7011)
```

Comment:
I think the above check will match `minimal.yml`, `minimal.yaml`, or even if the user has no minimal but only a `minimal.yml.bkp`, where the latter is a more plausible risk.

We can use some globbing inside `.glob`, like `y*ml`, but that could match other things too. Maybe the simpler would be to just do something like,

```python
if Path(conf_dir, 'minimal.yml').exists() or Path(conf_dir, 'minimal.yaml').exists():`
```

And maybe add an `an` in "Cannot copy experiment", so that it reads "Cannot copy an experiment ....".